src/Analysis.php

```diff
@@ -334,7 +333,7 @@ public function split()
         $result->merged = $this->merged();
         $result->unmerged = new Analysis();
         foreach ($this->annotations as $annotation) {
-            if ($result->merged->annotations->contains($annotation) === false) {
+            if ($annotation instanceof OpenApi === false && $result->merged->annotations->contains($annotation) === false) {
```

Comment:
This is because we cannot check different instances of `OpenAPI` now to be in `annotations` storage directly. Not sure if it is correct fix, but it helped